Missing app-lifecycle appStarting method

When running locally it throws this error:

```
Caused by: java.lang.UnsupportedOperationException: appStarting (com.github.clojure-lsp.intellij.listener.app-lifecycle/-appStarting not defined?)
        at com.github.clojure_lsp.intellij.listener.AppLifecycleListener.appStarting(Unknown Source)
        at com.intellij.util.messages.impl.MessageBusImpl.invokeMethod(MessageBusImpl.java:645)
        at com.intellij.util.messages.impl.MessageBusImpl.invokeListener(MessageBusImpl.java:624)
        at com.intellij.util.messages.impl.MessageBusImpl.deliverMessage(MessageBusImpl.java:417)
        at com.intellij.util.messages.impl.MessageBusImpl.pumpWaitingBuses(MessageBusImpl.java:390)
```

--- a/src/main/clojure/com/github/clojure_lsp/intellij/listener/app_lifecycle.clj
+++ b/src/main/clojure/com/github/clojure_lsp/intellij/listener/app_lifecycle.clj
@@ -10,6 +10,7 @@
 
 (defn -welcomeScreenDisplayed [_])
 (defn -appStarted [_])
+(defn -appStarting [_ _])
 (defn -projectFrameClosed [_])
 (defn -projectOpenFailed [_])
 (defn -appClosing [_])

